lib/ui/window/window.cc

```diff
@@ -249,6 +286,33 @@ void Window::UpdateAccessibilityFeatures(int32_t values) {
 }
 
 void Window::DispatchPlatformMessage(fml::RefPtr<PlatformMessage> message) {
+#if defined(OS_FUCHSIA)
```

Comment:
This can be handled in the platform view subclass in the Flutter runner. This would remove the need to have Fuchsia specific code in the engine.

Reply:
Nice!  Yeah, I was kinda unclear on which object owned which so this was a naive attempt.  I'll move it all to PlatformView in flutter_runner

Reply:
Fixed by using thread locals in SceneHost